client/src/pages/CourseList.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import Conditioning from "../assets/images/conditioning.JPG";
+import Pilates from "../assets/images/pilates.JPG";
+import WaterFitness from "../assets/images/waterfitness.JPG";
+import Meditation from "../assets/images/meditation.JPG";
+import NutritionHealth from "../assets/images/nutritionhealth.JPG";
+
+function CourseList() {
+    return (
+        <div>
+            <div className="about d-flex h-100 m-3 p-4">
+                <div className="p-3 text-left">
+                    <h2 className="mb-4">Conditioning and Flexibility</h2>
+                    <h3>
+                    This course provides knowledge, skill, and application of methods to maintain and improve the body's overall aerobic fitness and flexibility. 
```

Comment:
This course provides knowledge, skill, and application of methods to maintain and improve the bodies overall aerobic fitness and flexibility.